copy_assets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Define source and destination directories
+srcDir="src"
+destDir="built"
+
+# Function to copy .csv and .json files
+copyFiltered() {
```

Comment:
This files moves some files which we need runtime from the `src` directory to the `built` directory. Those are `*.csv` and `*.json` files which the Typescript compiler would ignore.